parameters/simulator/worlds/robotics_testlabs/objects_sdf.yaml

```diff
@@ -7,3 +7,6 @@
 - {id: coke-7, type: coke_can, x: 3.463, y: 4.353, z: 0.87}
 - {id: coke-under-table, type: coke_can, x: 0.4, y: 1.92, z: 0}
 - {id: person-1, type: person_standing, x: 0.833, y: 0.78, z: 0.0}
+- {id: coke-trash, type: coke_can, x: 1.8, y: 4.8, z: 0.53}
```

Comment:
These are coke cans added to test the take-out-the-garbage challenge. These can be removed

Reply:
Why? @KevinDang74 is that correct?